docs/rest.md

```diff
@@ -222,9 +226,10 @@ Sample result:
 
 #### Fields
 * `snaps`
-    * `status`: can be either `not installed`, `installed`, `active` (i.e. is
-      current), `removed` (but data present); there is no `purged` state, as a
-      purged snap is undistinguishable from a non-installed snap.
+    * `status`: can be either `buyable`, `available`, `installed`,
```

Comment:
Buyable is such a terrible term.. I couldn't come up with something better in our conversation, but now I actually have another suggestion for these sequence of events. Here is the full suggested text:

    * `status`: may transition as `available` => `installed` => `active`. For paid snaps,
       the initial state is `priced` and once bought it becomes `available`.